README.md

```diff
@@ -16,6 +16,11 @@ This is an Ember CLI addon, to install:
 
 `ember install ember-href-to`
 
+## Supported Ember Versions
+
+- v5.0.1. supports Ember versions >=3.1 & <3.27
+- It does not work in apps using Ember's modernized `LinkTo`, which was introduced in Ember 3.27
```

Comment:
Maybe give a little of reasoning why this doesn't work? You could also potentially link to the comment/conversation in the PR we just shipped?

>It does not work in apps using Ember's modernized `LinkTo`, which was introduced in Ember 3.27 and built using Glimmer components

Reply:
Will do it in a follow-up PR 👍